baal/bayesian/consistent_dropout.py

```diff
@@ -152,8 +172,10 @@ def __init__(self, module: torch.nn.Module):
                 A fully specified neural network.
         """
         super().__init__()
-        self.parent_module = module
-        _patch_dropout_layers(self.parent_module)
+        self.parent_module = patch_module(module)
 
     def forward(self, *args, **kwargs):
         return self.parent_module.forward(*args, **kwargs)
+
+    def unpatch(self) -> torch.nn.Module:
+        return unpatch_module(self.parent_module)
```

Comment:
this will make sure that `self.parent_module` is always patched but at every step the user can ask for an unpatched version right? its very useful just wanna make sure that if you call `unpatch` the internal module remains patched.

Reply:
yes